src/microbe_stage/Microbe.Contact.cs

```diff
@@ -1142,6 +1115,9 @@ private void HandleUnbinding()
     /// </summary>
     private void HandleEngulfing(float delta)
     {
+        if (!CanEngulf)
```

Comment:
I think this early returning is too early as this skips clearing `attemptingToEngulf` and also it doesn't handle the gradual smooth turning of the engulf audio. I think the cleanest refactor would be to create a new variable `actuallyEngulfing = State == MicrobeState.Engulf && CanEngulf` and replacing the existing checks in this method with that variable.

Reply:
Good idea, I'll fix this in a bit.